format_and_overlap_determination.md

```diff
@@ -0,0 +1,118 @@
+# Description of Format and Overlap Determinations #
+This attempts to describe what this system actually does. This does not attempt to exhaustively describe all corner cases or emergent properties.
+
+## Definitions ##
+**Format**
+: mono (spm), multi (mpm), serial (ser)
+
+**Cluster**
+: Collection of HT Items, Holdings, and Concordance pairs united by a list of OCNs determined to be equivalent.
```

Comment:
specify which system we're talking about, for example the very next definition (Bib record) refers to Zephir clustering.  If we're talking about both Zephir and PH for purposes of this definition, it might be worth saying that the logic/process for determining what is equivalent varies between systems.

Reply:
We are not talking about Zephir for the purposes of this definition.

Reply:
So is this better characterized as a "holdings cluster" or something?  Or alternatively, just take out the word "clustering" from the first sentence in the Bib Record definition below.

Reply:
Will change to 'preferred record from Zephir' and omit the reference to Zephir clustering.